model_analyzer/analyzer.py

```diff
@@ -136,6 +141,7 @@ def analyze(self):
         if not gpu_info:
             gpu_info = {}
         self._report_manager = ReportManager(
```

Comment:
Why do we need `ReportManager` in analyze mode?

Reply:
The analyze mode is what generates the summaries.